cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractSearchTest.java

```diff
@@ -60,6 +60,14 @@ public void shouldFindResourcesByTag() throws Exception {
         assertEquals(3, resources.size());
     }
 
+    @Test
+    public void shouldFindFolders() throws Exception {
+        cloudinary.api().createFolder("testFolder", null);
```

Comment:
I would make the folder name unique, and delete it on tearDown.

Reply:
Fixed